text/processing.py

```diff
@@ -70,19 +71,19 @@ def tasks(self):
 
     def run(self):
         pipeline = self.pipeline.copy()
-        if self.zip_out:
+        if self.zip_output:
             pipeline.append("gzip")
         self.pipe = " | ".join([str(i) for i in pipeline])
-        if isinstance(self.text, (list, tuple)):
-            self.input_text = " ".join(gs.file_caching(str(i)) for i in self.text)
+        if isinstance(self.text_files, (list, tuple)):
+            self.input_text = " ".join(gs.file_caching(str(i)) for i in self.text_files)
         else:
-            self.input_text = gs.file_caching(str(self.text))
-        self.sh("zcat -f {input_text} | {pipe} > {out}")
+            self.input_text = gs.file_caching(str(self.text_files))
+        self.sh("zcat -f {text_file} | {pipe} > {out}")
```

Comment:
here the input_text here refered to a member variable, now would be a new local var, probably better to insert via explicit string formatting